src/test/kotlin/dev/fakek/FakeContextTest.kt

```diff
@@ -88,6 +88,14 @@ internal class FakeContextTest {
         expectThat(fakeCompany).hasSize(1)
     }
 
+    @Test
+    fun `given a FakeContext when fakeCommerce is accessed multiple times it should return the same value multiple times`() {
+        val fakeCommerce = createDistinctList { subject.fakeCommerce }
+
+        expectThat(fakeCommerce).hasSize(1)
+    }
+
+    @Test
```

Comment:
@CodyEngel when I passed by this test file, I noticed that this spec had no @Test notation so, it would never run

also, with the ```expectThat```used in here https://github.com/CodyEngel/fakek/pull/31/files#diff-85a630ce2ccb24dae5169762179cad65R42 it is possible to avoid this kind of mistakes by merging all these samples in a single one

Reply:
Oof, good catch, thanks @RafaelPereiraSantos.

Reply:
Let's keep these tests separate for now, I worry it could become confusing since `FakeContext` will cover a variety of `Fake` classes. The test itself would always be changing so I'd be more comfortable creating new tests, I just need to make sure I drink morning coffee before approving PRs 😉

Reply:
If you think so, it is fine for me